static/js/pages/landing.js

```diff
@@ -41,38 +36,34 @@ Overview.prototype.zeroPadTotals = function() {
   helpers.zeroPad(this.selector + ' .js-snapshot', '.snapshot__item-number', '.figure__decimals');
 };
 
-$.getJSON(raisingUrl).done(function(data) {
-  var raisingData = [];
-
-  _.each(_.groupBy(data.results, 'date'), function(dateGroup) {
-    _.each(dateGroup, function(object) {
-      object[object.type] = object.receipts;
-      delete object.cycle;
-      delete object.receipts;
-      delete object.type;
-    });
-    raisingData.push(_.extend.apply(null, dateGroup));
-  });
-
-  var sorted = _.sortBy(raisingData, 'date');
-  new Overview('.js-raised-overview', sorted, 1);
-});
-
-$.getJSON(spendingUrl).done(function(data) {
-  var spendingData = [];
-
-  _.each(_.groupBy(data.results, 'date'), function(dateGroup) {
-    _.each(dateGroup, function(object) {
-      object[object.type] = object.disbursements;
-      delete object.cycle;
-      delete object.disbursements;
-      delete object.type;
-    });
-    spendingData.push(_.extend.apply(null, dateGroup));
+$.getJSON(entityTotalsURL).done(function(data) {
+  var spent = [];
+  var raised = [];
+  var sortedRaised;
+  var sortedSpent;
+
+  _.each(data.results, function(object) {
+    var raisedDatum = {
+      'date': object.date,
+      'candidate': object.cumulative_candidate_receipts,
+      'pac': object.cumulative_pac_receipts,
+      'party': object.cumulative_party_receipts
+    };
+    var spentDatum = {
+      'date': object.date,
+      'candidate': object.cumulative_candidate_disbursements,
+      'pac': object.cumulative_pac_disbursements,
+      'party': object.cumulative_party_disbursements
+    };
+
+    raised.push(raisedDatum);
+    spent.push(spentDatum);
   });
 
-  var sorted = _.sortBy(spendingData, 'date');
-  new Overview('.js-spent-overview', sorted, 2);
+  sortedRaised = _.sortBy(raised, 'date');
```

Comment:
Can the declaration be in this line `var sortedRaised = _.sortBy(raised, 'date')` instead of in the ajax request? Same for `sortedSpent`.